### ISSUE TITLE
Configure email settings

### ISSUE THREAD

**Issue by tituspijean** [YunoHost-Apps/flarum_ynh#22]
Populate Flarum's email settings with YunoHost's ones.

**Comment by tituspijean** [comment]
These settings require a user's password.
Should we 

- create a dedicated `flarum` user for such purpose ?
- use the designated admin's password ?
  - ask for it in the pre-installation form ?
  - leave it blank for the admin's to fill after installation ?

**Comment by tituspijean** [comment]
Email settings can be set by updating the database. See https://discuss.flarum.org/d/620-smtp-setup/10 :
 
```
INSERT INTO `config`(`key`, `value`) VALUES
('mail_host', 'mail.sample.com') ,
('mail_port', '25'),
('mail_username', 'youusername'),
('mail_password', 'secret'),
('mail_encryption', 'ssl')
```

**Comment by tituspijean** [comment]
After manually configuring emails in the admin panel, it seems the settings have moved within the db. These are the settings enough to send emails.

```
INSERT INTO `settings` (`key`, `value`) VALUES
('mail_driver', 'mail'),
('mail_encryption', 'ssl'),
('mail_from', 'flarum@ynh.dev'),
('mail_host', 'localhost'),
('mail_port', '587');
```

I suggest to use `$app` instead of directly `flarum` to differenciate between instances, although `flarum__2` may not be quite comprehensive for the end user ?

**Comment by tituspijean** [comment]
Solved by #62.
